test/timestamp-nano.test.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+/* eslint no-prototype-builtins: 0 */
+
+const ms = 1754060611115
+const now = Date.now
+Date.now = () => ms
+
+const hrTimeBigint = process.hrtime.bigint
+process.hrtime.bigint = () => 101794177055958n
```

Comment:
Can you avoid this top level changes? This would make the test unmaintanable.

Reply:
they're run in isolation though, aren't they?